README.md

```diff
@@ -148,18 +149,22 @@ jupyter notebook
 --->
 
 # <a name="models"></a>
+# Supported Models
 
-# Models
+Any models that are trained on datasets in Python `numpy.array`, `pandas.DataFrame`, `iml.datatypes.DenseData`, or `scipy.sparse.csr_matrix` format are supported by this API.
```

Comment:
It might be a bit confusing to say "Any models that are trained on datasets...are supported by this API" and then say in the next paragraph that we only work for models that provide `predict` or `predict_proba`. Maybe "This API supports models that are trained on datasets in Python [datatypes] format."?